src/generator/protocol/operations.ts

```diff
@@ -196,15 +285,15 @@ function createProtocolRequest(client: string, op: Operation, imports: ImportMan
     // add specific request headers
     const headerParam = values(op.request.parameters).where((each: Parameter) => { return each.protocol.http!.in === 'header'; });
     headerParam.forEach(header => {
-      text += `\treq.Header.Set("${header.language.go!.name}", ${formatParamValue(header, imports)})\n`;
+      text += `\treq.Header.Set("${header.language.default.name}", ${formatParamValue(header, imports)})\n`;
```

Comment:
Might want to add a comment why the default name is used here.

Reply:
Done